tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
+
+\noindent\textbf{Reward:} 750 points (250 points per guest).
```

Comment:
Check if max can be 500 with option to raise it via bonuses. I would reduce it to 150pts per guest, providing only location.

Reply:
In the meeting we said 750 points for the main goals and 250 for the bonuses for all tests...